Game/Assets/Coliseo/Scripts/Enemy.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UnityEngine;
+using System.Collections;
+using Coliseo;
+using System;
+
+namespace Coliseo
+{
+    public class Enemy : Actor
+    {
+
+        private const float MIN_DIST = 1.5f;
```

Comment:
Quite frankly, I just picked them because they visually looked nice when playing. But I'll add a comment and possibly rename them a tad.